src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientManager.cs

```diff
@@ -334,5 +339,27 @@ private async Task RefreshFallbackClients(CancellationToken cancellationToken)
 
             _lastFallbackClientRefresh = DateTime.UtcNow;
         }
+
+        private async Task<bool> IsValidEndpoint(string hostName)
+        {
+            if (string.IsNullOrWhiteSpace(hostName))

```

Comment:
Why would we have null or whitespace here?

Reply:
Explained in another comment